__tests__/unittest/__snapshots__/templates.test.js.snap

```diff
@@ -54,91 +54,18 @@ exports[`templates createAPIResourceTemplate should create API resource template
 `;
 
 exports[`templates createEventResourceTemplate should create event resource template correctly 1`] = `
-{
```

Comment:
@zongqichen : This snapshot is re-generated(by using `npx jest -u`. Does it seem correct to you? Not sure if this is the command that should be used or I've missed something?

Reply:
Re-generated is fine, but we need to remove the visibility from extensions. Otherwise, we will lost the test. I would suggest add new testcase, e.g. `it(should filter out not public object)`

Reply:
@zongqichen :
4 new assertions are added into the templates.test.js: for private and internal visibility of apiResources and apiEvents.
The check if both functions(getApiResources and getEventResources) are returning arrays are added as well.
Reminder: before, the function getEventResources was returning an object; this is wrong because it should return **single-item array**.
cc: @Fannon, @swennemers